tests/acceptance/29_simulate_mode/kept-in-manifest.cf

```diff
@@ -0,0 +1,48 @@
+body common control
+{
+      inputs => {
+        "../default.cf.sub",
+        "./prepare_files_for_simulate_tests.cf.sub",
+        "./normalize_agent_output.cf.sub"
+      };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+
+bundle agent init
+{
+  methods:
+    "prepare_files_for_simulate_tests";
+}
+
+bundle agent test
+{
+  meta:
+    "test_soft_fail" string => "any",
+      meta => { "CFE-3506" },
+      comment => "already-created file is created before agent run but not copied into chroot and not included in manifest";
```

Comment:
I believe it is actually copied into the chroot.

Reply:
Mm. Right. The ticket is about the copied file not being part of the manifest so I can change that description as I understood it differently when I wrote that. :+1: